pac4j-oidc/src/main/java/org/pac4j/oidc/credentials/extractor/OidcExtractor.java

```diff
@@ -99,6 +99,11 @@ public OidcExtractor(final OidcConfiguration configuration, final OidcClient cli
             logger.debug("Authentication response successful");
             var successResponse = (AuthenticationSuccessResponse) response;
 
+            if (configuration.getProviderMetadata().supportsAuthorizationResponseIssuerParam() &&
```

Comment:
Should we check nullity on the `successResponse` or `getIssuer()`?

Can you create a variable for the `configuration.getProviderMetadata()` part outside the `if`? Thanks

Reply:
The response cannot be null. If the response cannot be parsed, the parse method will throw an exception. The issuer is a required parameter for the metadata, the code will refuse to instantiate a `AuthorizationServerMetadata` with a `null` issuer. When the server advertises support for this parameter, it is required to set it. Failing to so do must be treated as a mismatch, that's why the equals is written like this: metadata issuer is guaranteed to be not null, response issuer must be not null to match.

I'll update the code to extract a local var for the metadata and rebase on master.